scripts/run_vsim.sh

```diff
@@ -56,6 +56,49 @@ exec_test() {
                 call_vsim tb_axi_atop_filter -GN_TXNS=1000 -GAXI_MAX_WRITE_TXNS=$MAX_TXNS
             done
             ;;
+        axi_iw_converter)
+            for SLV_PORT_IW in 1 2 3 4 8; do
+                MAX_SLV_PORT_IDS=$((2**SLV_PORT_IW))
+                MAX_UNIQ_SLV_PORT_IDS_OPTS=(1 2)
+                if [ $MAX_SLV_PORT_IDS -gt 2 ]; then
+                    MAX_UNIQ_SLV_PORT_IDS_OPTS+=(3 4)
+                fi
+                if [ $(($MAX_SLV_PORT_IDS/2)) -ge 4 ]; then
+                    MAX_UNIQ_SLV_PORT_IDS_OPTS+=($((MAX_SLV_PORT_IDS/2-1)))
+                fi
+                MAX_UNIQ_SLV_PORT_IDS_OPTS+=($MAX_SLV_PORT_IDS)
+                for MST_PORT_IW in 1 2 3 4; do
+                    if [ $MST_PORT_IW -lt $SLV_PORT_IW ]; then # downsize
+                        for MAX_UNIQ_SLV_PORT_IDS in "${MAX_UNIQ_SLV_PORT_IDS_OPTS[@]}"; do
+                            MAX_MST_PORT_IDS=$((2**MST_PORT_IW))
+                            if [ $MAX_UNIQ_SLV_PORT_IDS -le $MAX_MST_PORT_IDS ]; then
+                                call_vsim tb_axi_iw_converter \
+                                    -t 1ns -coverage -classdebug -voptargs="+acc +cover=bcesfx" \
+                                    -GAxiSlvPortIdWidth=$SLV_PORT_IW \
+                                    -GAxiMstPortIdWidth=$MST_PORT_IW \
+                                    -GAxiSlvPortMaxUniqIds=$MAX_UNIQ_SLV_PORT_IDS \
+                                    -GAxiSlvPortMaxTxnsPerId=5
```

Comment:
Thanks for the hint! That is good to know and unexpected for me. (I'm sure there is an arcane use case for overriding all parameters, but making this the default behavior? :thinking: )

I think a simple, generic `Tb` prefix should suffice, since TBs in this repo are not nested. Do you agree?

Reply:
There was another option in the tool that targets only the top-level of the design. But I can't remember exactly what it was -- maybe something like `-g`.